apps/src/code-studio/components/AssetUploader.jsx

```diff
@@ -24,14 +27,19 @@ var AssetUploader = React.createClass({
   },
 
   render: function () {
+    let api = this.props.useFilesApi ? filesApi : assetsApi;
+    let url = api.getUploadUrl();
+    let uploadDone = api.makeUploadDoneCallback(this.props.onUploadDone);
+    let uploadStart = api.makeUploadStartCallback(this.props.onUploadStart);
```

Comment:
good suggestion on rename. Not sure how to check that we're not forgetting going forward, but for now, `HiddenUploader` is only used by `AssetUploader` and `AnimationPicker`. I didn't update the latter since we're not yet changing GameLab.